### PR TITLE
Issue/4296 reader calypso site preview

### LINKED ISSUE #4296: Reader: site preview doesn't match Calypso
The site preview on Android should be adjusted to match Calypso:
* Follower count should appear below the description
* Site preview header should include blavatar
* Blavatar and header should be removed from individual posts

Calypso:

calypso-followers

Android:

android-followers

### PR DESCRIPTION
**Pull request by nbradbury**
Fixes #4296 - reader site preview now more closely matches the Calypso site preview. Note that minor differences remain, most obviously the fact that on Android the dateline below the post title doesn't contain three tappable labels (instead it's just one untappable date label, since the tap targets would be too small).

To test: Tap the header above a wp.com post to view the site preview.

device-2016-06-25-091844

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/reader/adapters/ReaderPostAdapter.java [hunk 1]
```diff
@@ -194,6 +210,13 @@ public GapMarkerViewHolder(View itemView) {
         }
     }
 
+    /*
+     * post header isn't shown for blog preview
+     */
+    private boolean shouldShowPostHeader() {
```
Comment (hypest) [inline WordPress/src/main/java/org/wordpress/android/ui/reader/adapters/ReaderPostAdapter.java]: Hey @nbradbury , this looks pretty much as the inverse of `isBlogPreview()`. May I suggest dropping is in favour of `isBlogPreview()`? It seems that there are quite a few comments around that will show what happens with the header when it is a blog preview.

File: WordPress/src/main/res/layout/reader_blog_info_view.xml [hunk 2]
```diff
@@ -78,6 +79,37 @@
             app:wpFontFamily="merriweather"
             tools:text="text_description" />
 
-    </FrameLayout>
+        <LinearLayout
```
Comment (hypest) [inline WordPress/src/main/res/layout/reader_blog_info_view.xml]: Unfortunately, nesting the followers count inside the `layout_blog_description` hides it when the blog has no description. Calypso seems to show the followers count even in that case.

File: WordPress/src/main/res/layout/reader_blog_info_view.xml [hunk 3]
```diff
@@ -20,10 +20,18 @@
         android:paddingTop="@dimen/margin_large"
         tools:targetApi="LOLLIPOP">
 
+        <org.wordpress.android.widgets.WPNetworkImageView
+            android:id="@+id/image_blavatar"
+            style="@style/ReaderImageView.Avatar"
+            android:layout_centerVertical="true"
+            android:layout_marginRight="@dimen/margin_large"
+            tools:src="@drawable/blavatar_placeholder" />
+
```
Comment (hypest) [inline WordPress/src/main/res/layout/reader_blog_info_view.xml]: Not exactly an issue of this PR but, now that there is extra space used by the blavatar, would you consider anchoring the `text_blog_name` and `text_domain` to the left of the `follow_button`? As it is, a big name (and/or a big system font setting) can make these texts collide with the follow button. There _is_ such anchoring in the `layout/reader_cardview_post.xml` case.

**Comment by hypest** [comment]
Hey @nbradbury , I've finished my pass on the PR. The only blocker for me is the hiding of the followers count which I think happens accidentally. Let me know if the hiding is by design though. Thanks!

File: WordPress/src/main/java/org/wordpress/android/ui/reader/adapters/ReaderPostAdapter.java (hunk 1 above)
Comment (nbradbury) [inline WordPress/src/main/java/org/wordpress/android/ui/reader/adapters/ReaderPostAdapter.java]: A fair point - corrected in b785879.

**Comment by nbradbury** [comment]
> The only blocker for me is the hiding of the followers count which I think happens accidentally.

Good catch! Yes, that's accidental - corrected in 795114d.

File: WordPress/src/main/res/layout/reader_blog_info_view.xml (hunk 3 above)
Comment (nbradbury) [inline WordPress/src/main/res/layout/reader_blog_info_view.xml]: Makes sense. Added in b7b88d6.

**Comment by nbradbury** [comment]
@hypest Reader for another review, thanks!

**Comment by hypest** [comment]
🚢 it!

**Pull request merged**